testing/src/uploadResultTQ.js

```diff
@@ -0,0 +1,58 @@
+var request = require('request');
+var fs = require('fs');
+import * as dotenv from "dotenv";
+
+
+
+token = undefined
+const tq_username = process.env.TQ_USER_NAME;
+const tq_password = process.env.TQ_PASSWORD;
+const tq_secret = process.env.TQ_SECRET;
+var options = {
```

Comment:
The other projects are using Axios to execute the HTTP requests, maybe it would be good to follow the same.